doc/03_reference/instruction_decode_execute.rst

```diff
@@ -66,15 +66,15 @@ Other blocks use the ALU for the following tasks:
 
 Bit-Manipulation Extension
   Support for the `RISC-V Bit-Manipulation Extension version 1.0.0 <https://github.com/riscv/riscv-bitmanip/releases/download/1.0.0/bitmanip-1.0.0-38-g865e7a7.pdf>`_ and `draft version 0.93 from January 10, 2021 <https://github.com/riscv/riscv-bitmanip/blob/master/bitmanip-0.93.pdf>`_ is optional. [#B_draft]_
-  It can be enabled via the enumerated parameter ``RV32B`` defined in :file:`rtl/cve2_pkg.sv`.
+  It can be enabled via the enumerated local parameter ``RV32B`` defined in :file:`rtl/cve2_pkg.sv`.
```

Comment:
we should say that we as OpenHW Group do not maintain the RVB extensions, therefor users are responsible to fork the core and enable the parameter at their own risk in their fork, so that it is clear that these ISA extensions won't be at high TRL, nor maintained by OpenHW Group

Reply:
done